packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -998,6 +998,30 @@ export function getProfilesAndStatusesForPosts(postsArrayOrMap: Post[]|Map<strin
     return Promise.all(promises);
 }
 
+export function getPostsByIds(ids: string[]) {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        let posts;
+
+        try {
+            posts = await Client4.getPostsByIds(ids);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
+                {type: PostTypes.GET_POSTS_FAILURE, error},
```

Comment:
I think we can omit this action. We used to keep track of a bunch of request state with actions like these, but most of it isn't used any more.

I've got a ticket around to go clean that up eventually since we started removing it at some point before we kind of forgot about it.